src/components/pages/Playground/Playground.module.css

```diff
@@ -9,6 +9,9 @@
 .buttonContainerColor {
   background-color: #012f34;
 }
+.buttonContainerAlternateColor {
+  background-color: white;
```

Comment:
I know that the Playground stuff is just for experimenting and figuring out how things work, but since you're touching color stuff here, I think if you can convert the colors to use variables where we've implemented them already, it would help demonstrate the use of CSS variables for our teammates. so line 10, `#012f34` would convert to `var(--tertiary-color)`. And check for other colors that have existing vars.